composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/presentation/CreateEventViewModel.kt

```diff
@@ -226,6 +252,63 @@ class CreateEventViewModel(dispatcherProvider: DispatcherProvider, savedStateHan
         }
     }
 
+    private fun onAttendeeEmailChange(text: String) {
+        if (viewStateFlow.value.isAttendeeEmailError) {
+            emitViewState { viewState ->
+                viewState.copy(
+                    isAttendeeEmailError = validateEmailUseCase(text)
+                )
+            }
+        }
+    }
```

Comment:
Don't you need to update the text state as well? Or are you working with `TextFieldState`?